advance_check_void/__manifest__.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (C) 2019 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Advance Check Void",
+    "version": "14.0.1.0.0",
+    "category": "Accounting",
+    "summary": "Void History for Check Payments",
+    "author": "Open Source Integrators",
+    "website": "https://github.com/ursais/osi-addons",
+    "license": "AGPL-3",
+    "depends": [
+        "account_accountant",
```

Comment:
Can we lower the dependency requirements?
```suggestion
        "account",
```